src/lib/addons/__snapshots__/slack.test.ts.snap

```diff
@@ -1,9 +1,18 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Should call slack webhook 1`] = `"{"username":"Unleash","icon_emoji":":unleash:","text":"some@user.com created feature toggle <http://some-url.com/projects/default/features/some-toggle|some-toggle> in project *default*","channel":"#undefined","attachments":[{"actions":[{"name":"featureToggle","text":"Open in Unleash","type":"button","value":"featureToggle","style":"primary","url":"http://some-url.com/projects/default/features/some-toggle"}]}]}"`;
+exports[`Should call slack webhook 1`] = `"{"username":"Unleash","icon_emoji":":unleash:","text":"some@user.com created feature toggle <http://some-url.com/projects/default/features/some-toggle|some-toggle> in project *default*","channel":"#general","attachments":[{"actions":[{"name":"featureToggle","text":"Open in Unleash","type":"button","value":"featureToggle","style":"primary","url":"http://some-url.com/projects/default/features/some-toggle"}]}]}"`;
```

Comment:
Have you fixed slack integration by chance?

Reply:
No, have not changed it, just given it support for custom headers

Reply:
It changed from `undefined` to `general` so the support for custom headers might have helped. Not sure but :+1:

Reply:
Oh, that's just because I'm now type checking the parameters, so our test no longer allowed us to activate the slack addon with no defined defaultChannel (it's marked as required in our definition). Earlier, the test that wrote this snapshot did not have the defaultChannel parameter set, which caused javascript to happily send `undefined` :)